packages/components/src/vue/components/morph-parts/principal-parts.vue

```diff
@@ -6,13 +6,17 @@
         <h4
             class="alpheios-principal-parts__groupitem alpheios-principal-parts__groupitem--lemma-word"
             :lang="languageCode"
+            data-alpheios-enable="all"
             v-if="! lemma.principalParts.includes(lemma.word)"
-        >{{lemma.word}}</h4>
+        ><span
+          class="alpheios-principal-parts__groupitem--lemma-word__listitem"
+          v-for="word in lemma.word.split(/[,-]/)">{{word}}</span></h4>
```

Comment:
I think it might probably be better, rather than splitting in place, create a method on the Vue instance and handle it there. I expect after running it in the wild we might run into some not-so-trivial splitting cases that might require more complex conditions and error handling. Such logic better be separated into its own method, I think. Maybe it would even deserve a method in the `Lemma` class (but not now, of course). What do you think?

Reply:
ok, you're probably right about making a method.

In terms of where it belongs in the long run, as I alluded to in #135, I think the query code needs handle this and create a data model object that more accurately reflects what this is -- i.e a compound word with multiple lemmas.